.pre-commit-config.yaml

```
@@ -15,7 +15,7 @@ repos:
   rev: v2.2.6
   hooks:
   - id: codespell
-    args: ["-L", "ned,ist,oder"]
+    args: ["-L", "ned,ist,oder","--skip","*.po"]
```

The littlest of nits, otherwise looks good 👍 

```suggestion
    args: ["-L", "ned,ist,oder", "--skip", "*.po"]
```

Oh, and did the ned, ist and/or oder come from a po file? If so, we can remove those from this config too.